external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -247,6 +247,32 @@ class AvroSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
     }
   }
 
+  test("Union type: More than one non-null type") {
```

Comment:
I filed a JIRA for you. Usually, we create a JIRA first and make a PR to that.
- https://issues.apache.org/jira/browse/SPARK-27858

A test case of BUG fix usually have the prefix of JIRA ID like the following. Please update the test case name accordingly.
```
test("SPARK-27858 Union type: More than one non-null type") {
```

Reply:
Also done!